lib/network_utils.pm

```diff
@@ -65,6 +65,11 @@ Main use case is post_fail_hook, to be able to upload logs
 sub recover_network {
     my (%args) = @_;
 
+    if ($args{reset_network}) {
+        script_run('rm /etc/sysconfig/network/ifcfg-*');
```

Comment:
not sure about this, this will remove also ifcfg-lo.
Comment from Marius in #wicked:

```
11:41 first of all, make sure there is a correct ifcfg-lo with STARTMODE=nfsroot, aka aviod rm -rf ifcfg-*
11:42 -> ipv6 for eth0 requires lo too -- there are several things registered for lo as a global system property
```

Reply:
still I updated logic to a smarter one